### ISSUE TITLE
bug : accessibility warning

### ISSUE THREAD

**Issue by chaesunbak** [chaesunbak/spoti-guessr2#1]
990-3f3dadbd17bc543e.js:1 Warning: Missing `Description` or `aria-describedby={undefined}` for {DialogContent}.
